### DISCUSSION TITLE
Practica 1 ayuda por favor

### DISCUSSION THREAD

**Discussion by MdLuffy1301** [Launch-X-Latam/MisionFrontEnd#135] [Q&A]
Podrían darme su opinion sobe mi wireframe esta en mi repositorio con el nombre My Firts Board, la verdad estaba muy perdido ya que por el trabajo apenas pude ver la clase

**Comment by Evelo00** [comment]
Hola! revisa un poco las imagenes, no se si es cosa mia, pero las veo con mala calidad. en si esta muy bien. me gusto tu analisis de requisitos, corto y claro.

**Reply by MdLuffy1301** [reply]
gracias no sabes como me alienta eso, en cuanto a la calidad creo que se debe al archivo , te dejo el link https://miro.com/app/board/uXjVPQjK8ac=/?share_link_id=217158326332

**Comment by AndrjsM** [comment]
Hola, muy buen diseño, yo en lugar de "en proceso" podría "Mis solicitudes" ya que el estado de las solicitudes puede ser multiple. 

Agregaría opciones de registro como Google para facilitar el proceso. 

Pero en general muy buen diseño.

**Comment by LuceroLuciano** [comment]
Vas muy bien Explorer! 

Saludos.

**Comment by Wyspect** [comment]
Creo que el wireframe esta bien, pero estas omitiendo algo importante de las instrucciones 

> Es un despacho de abogados que quiere automatizar las demandas de sus clientes, esto lo harán a traves de una página web llenando un formulario.

Es decir que la práctica la haremos pensando en una página web y veo que tu wireframe es más parecido a una app movil

**Reply by Evelo00** [reply]
buenas, no del todo. tiene que ser responsive para app movil tambien, y se nos dijo que era "buena practica" empezar por el diseño de menor tamaño. toca hacer ambos igual. de celular y de pagina web de pc

**Reply by LauraHenao27** [reply]
Es verdad, es más facil hacer responsive algo de menor dimensión, para no crear conflicto si lo haces primero en página web

**Comment by dmejia96** [comment]
Hola, me gustó tu información, muchas felicidades!

**Comment by ElPoder1** [comment]
Buenas, lo veo muy bien, creo que esa etapa es mas para hacer un borrador que nos de la idea y ya al final hacer el Wireframe de UI ya con diseño en Figma
